### ISSUE TITLE
Dynamic favicon

### ISSUE THREAD

**Issue by ewels** [seqeralabs/nf-tower#231]
It would be a nice touch if the browser favicon could change dynamically according to what's going on in the page.

For example - the current black favicon could be the default, but it could switch to blue / green / red to match the status of the current workflow shown in the page.

I think it should be relatively easy - just a case of editing the HTML DOM attribute as [described here](https://www.geeksforgeeks.org/how-to-change-favicon-dynamically/).

**Comment by evanfloden** [comment]
Very cool idea. I notice GH does this for checks.

<img width="52" alt="Screenshot 2020-05-27 08 22 48" src="https://user-images.githubusercontent.com/3192944/82984943-634bc200-9ff3-11ea-8944-715298a89a25.png">

**Comment by ewels** [comment]
Yup, and I think Travis used to do it - Gmail does it for me, it's a really nice touch. Especially for those amongst us who like to have hundreds of tabs open 😅
